In [2]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt

# How many people go to the DMV?
http://answers.google.com/answers/threadview/id/531251.html

The number of DMV visits in the us per year: 178.8 million

Cook county population/us population: 5.15/329.5 = .0156

num dmvs in cook county: 9

In [18]:
# Approx people per day at the dmv
bussiness_days_year = 260
num_per_day = (178.8*.0156/bussiness_days_year)/9 * 1000000
num_per_hour = num_per_day/9.5
num_per_hour

125.47368421052634

We would like to have an average of around 125 people visit the dmv per day. We also have 2 busy periods, one in the morning and one during lunch.

In [27]:
weights = [.0526,.3158,.3158,.3158]
rates = [160,110,160,110]
np.dot(weights,rates)

128.42000000000002

## Parameter Estimates
morning_rush_end = 0.5 # until 8:00

lunch_rush_start = 3.5 # 11:00 start

lunch_rush_end = 6.5 # 2:00 end

---

decreased_arrival_rate = 110

increased_arrival_rate = 160

# Why do people go to the DMV?
https://yogov.org/blog/most-popular-reasons-visit-dmv/

This data is from california.

| Percent | Reason                                                                       |
|---------|------------------------------------------------------------------------------|
| 11.9    | behind the wheel                                                             |
| 5.8     | transfer vechicle title                                                      |
| 15.3    | permit Test                                                                  |
| 25.5    | First license                                                                |
| 15.1    | Other (changing address, changing names, getting license plates, and others) |
| 13.7    | Renew License                                                                |
| 12.7    | Register Vehicle                                                             |

Condense

| Percent            | Reason                    | Actions needed |
|--------------------|---------------------------|----------------|
| 11.9               | Driving Test              | Camera,RT      |
| 5.8+15.1+12.7=33.6 | No Test                   |                |
| 25.5+15.3=40.8     | permit test/First license | Camera,WT      |
| 13.7               | Renew Licese              | Camera         |

## Parameter Estimates
p_checkin_camera: 11.9+40.8+13.7 = 66.4

p_checkin_clerk: 33.6

---
p_camera_clerk: 13.7/66.4 = 20.6

p_camera_roadtest: 11.9/66.4 = 18

p_camera_writtentest: 40.8/66.4 = 61.4